src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -90,7 +91,8 @@ def build_io_uris_ingestion(
 
         import tiledb
 
-        vfs = tiledb.VFS()
+        ctx = tiledb.Ctx(config)
+        vfs = tiledb.VFS(ctx=ctx)
```

Comment:
Right now this VFS instance is getting the `access_credentials_name` (given during the node creation) and is being initialized with the node's default context including these access_credentials. Would not this overwrite the context of the instance, leading to misconfiguration?

Reply:
Users shouldn't be passing credentials in to the config, only other settings like region. I don't think it'll be an issue for auth.